### PR TITLE
Track refactorings executed by `@bpmn-io/refactorings`

### PR DESCRIPTION
**Pull request by philippfromme**
Related to https://github.com/camunda/web-modeler/issues/9039

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        node-version: [14]
```
Comment (nikku) [inline .github/workflows/CI.yml:8]: As we're on this we could drop the Cache Node.js modules task, too, and add `cache: 'npm'` to the setup-node action ([ref](https://github.com/bpmn-io/bpmn-js/blob/develop/.github/workflows/CI.yml#L19)).

File: lib/features/refactorings/RefactoringsTracking.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    this._eventBus.on('refactorings.execute', this.trackRefactoring.bind(this));
+  }
+
+  trackRefactoring(event) {
+    const { refactoring } = event;
+
+    this._bpmnJSTracking.track({
+      name: 'refactorings.execute',
+      data: {
+        refactoring
```
Comment (nikku) [inline lib/features/refactorings/RefactoringsTracking.js:15]: Quick-check: Can this leak personally identifyable information?

File: test/spec/features/RefactoringsTracking.spec.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+      bpmnJSTracking.on('tracking.event', spy);
+
+      const [ refactoring ] = await refactorings.getRefactorings([ element ]);
+
+      // when
+      refactoring.execute();
+
+      // then
+      expect(spy).to.have.been.calledOnce;
```
Comment (nikku) [inline test/spec/features/RefactoringsTracking.spec.js:114]: Great we verify the contract (data shape), too!

**Review by nikku — APPROVED** [review]
:heavy_check_mark:

File: lib/features/refactorings/RefactoringsTracking.js (hunk 2 above)
Comment (philippfromme) [reply lib/features/refactorings/RefactoringsTracking.js:15]: We recently made a change to [sanatize all events](https://github.com/bpmn-io/bpmn-js-tracking/blob/main/lib/BpmnJSTracking.js#L70) so we don't have to do it manually in each handler.

File: .github/workflows/CI.yml [hunk 4]
```diff
@@ -16,15 +16,7 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - name: Cache Node.js modules
```
Comment (nikku) [inline .github/workflows/CI.yml:19]: Let's next time use the `ci` prefix for this:

```
ci: rework everything to run on Flips machine only
```

**Pull request merged**